plugin/core/windows.py

```diff
@@ -22,7 +26,7 @@
     assert LanguageConfig
     assert WorkspaceFolder
 except ImportError:
-    pass
+    from sublime_lib import OutputPanel
```

Comment:
Strange place for an import - is this where you are running into mypy errors?

Reply:
When I put it above, I get massive mypy errors. Got frustrated, gave up with sublime_lib.